packages/explorer-ui/constants/tokens/basic.ts

```diff
@@ -1081,8 +1173,14 @@ export const BASIC_TOKENS_BY_CHAIN = {
     ETH,
     NETH,
     WETH,
+    USDC,
```

Comment:
Probably don't need `USDC` here, as it doesn't have a `ChainId.BASE` key at this point